### DISCUSSION TITLE
Guide to Deploying Outline with Docker

### DISCUSSION THREAD

**Discussion by routerino** [outline/outline#2684] [Show and tell]
Now that outline offers generic OIDC support, I have rewritten my [outline deployment guide](https://blog.gurucomputing.com.au/doing-more-with-docker/deploying-outline-wiki/) to utilize [keycloak](https://www.keycloak.org/) as Outline's authentication mechanism.

This is also part of a larger guide on how to set up [a selfhosted docker environment](https://blog.gurucomputing.com.au/doing-more-with-docker/) if you're looking for a more general docker guide.

**Comment by johnoscott** [comment]
Nice work. Thanks for sharing this @routerino
